tests/cypress/e2e/systemNameReadOnlyInAreas.cy.ts

```diff
@@ -0,0 +1,71 @@
+import { createSite, createUser, deleteSite, deleteUser, grantRoles } from '@jahia/cypress'
+import { ContentEditor } from '../page-object/contentEditor'
+import { PageComposer } from '../page-object/pageComposer'
+import { CustomPageComposer } from '../page-object/pageComposerOverride'
```

Comment:
It's a bit weird to have a PageComposer and CustomPageComposer, can't we move the necessary function in PageComposer and remove the CustomPageComposer?